Source/WTF/wtf/FileSystem.cpp

```diff
@@ -836,6 +839,17 @@ String lexicallyNormal(const String& path)
     return fromStdFileSystemPath(toStdFileSystemPath(path).lexically_normal());
 }
 
+bool isAncestor(const String& possibleAncestor, const String& possibleChild)
+{
+    auto possibleChildLexicallyNormal = lexicallyNormal(possibleChild);
+    auto possibleAncestorLexicallyNormal = lexicallyNormal(possibleAncestor);
+    if (possibleChildLexicallyNormal.endsWith(std::filesystem::path::preferred_separator))
+        possibleChildLexicallyNormal = possibleChildLexicallyNormal.left(possibleChildLexicallyNormal.length() - 1);
+    if (possibleAncestorLexicallyNormal.endsWith(std::filesystem::path::preferred_separator))
```

Comment:
`if (possibleAncestorLexicallyNormal.endsWith(static_cast<char>(std::filesystem::path::preferred_separator)))` to fix the Win build.